lib/src/dataflow/tuple.dart

```diff
@@ -66,7 +66,7 @@ class Aes {
   final Label? label;
 
   /// The size of the tuple.
-  final double? size;
+  double? size;
```

Comment:
Needed to remove the `final` keyword from this property so that it can be changed from a modifier. 
This is [similar to `ggplot`'s `dodge2` modifier ](https://ggplot2.tidyverse.org/reference/position_dodge.html#ref-examples)that also changes the size of its elements